.Rprofile

```diff
@@ -0,0 +1,4 @@
+if (Sys.getenv("USE_RENV", unset = "TRUE") == "TRUE") {
```

Comment:
is this some form of high level check of environmental variable? 
Was it initialised automatically by creating renv? I think that for now you should jest setup renv and update it with new package added. I'm not quite sure if complicated workflow with package checks is the best idea at this point

Reply:
Basically, installing renv broke CI jobs (CDM checks and test coverage) - this code is to overcome it and fix checks, it is based on @krystian8207 's commit activity on DataFakeR.